survos/command-bundle/1.5/config/routes/survos_command.php

```diff
@@ -0,0 +1,12 @@
+<?php
+// config/routes.php
+
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+return static function (RoutingConfigurator $routes): void {
+    $routes->import('@SurvosCommandBundle/config/routes.php')
+        ->prefix('/admin') // consider adding this route to the access_control key in security
+        ->schemes(['https'])
```

Comment:
We don't usually enforce https at this level. The user should definitely have this, but it's on them.